sass/_footer.scss

```diff
@@ -0,0 +1,71 @@
+.footer {
+  right: 0;
+  bottom: 0;
+  left: 0;
+  padding: 10px;
+  background-color: #053352;
+  color: #cccccc;
```

Comment:
you've used `#CCCCCC` a lot in your files -- maybe would make sense to make a variable out of it to use?